src/client/src/App.css

```diff
@@ -0,0 +1,77 @@
+* {
```

Comment:
These css look rather strange. Do you really need all those `1px` and other hardcoded pixels?